packages/@react-spectrum/contextualhelp/src/contextualhelp.css

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+:root {
+  --spectrum-contextualhelp-button-size: var(--spectrum-global-dimension-size-250);
+  --spectrum-contextualhelp-icon-size: var(--spectrum-global-dimension-size-175);
+  --spectrum-contexualhelp-dialog-width: 250px;
```

Comment:
I still need to verify this size. XD UI kit wasn't clear if the popover width be fixed. Looks like the width should be smaller than a small dialog however.

Reply:
2352-contextual-help

> Cierra [# 2352](https://github.com/adobe/react-spectrum/issues/2352)
> 
> ##  Lista de verificación de solicitud de extracción:
> * [ ] Enlace incluido al [problema de GitHub de React Spectrum](https://github.com/adobe/react-spectrum/issues) correspondiente .
> * [ ]  Se agregaron / actualizaron pruebas unitarias y un libro de cuentos para este cambio (para un nuevo código o código que ya tiene pruebas).
> * [ ]  Instrucciones de prueba completas.
> * [ ]  Documentación actualizada (si ya existe para este componente).
> * [ ] Consulte las prácticas de accesibilidad para esta función: [prácticas de Aria](https://www.w3.org/TR/wai-aria-practices-1.1/)
> 
> ##  Instrucciones de prueba:
> ![image](https://user-images.githubusercontent.com/1004204/142924791-9950e5c1-076c-44bc-bb9a-3d27d71ff201.png)
> 
> Libro de cuentos: https://reactspectrum.blob.core.windows.net/reactspectrum/c61511f17f3a55ac44dcb2ce9b104373f272fb60/storybook/index.html?path=/story/contextualhelp--default



> Cierra [# 2352](https://github.com/adobe/react-spectrum/issues/2352)
> 
> ##  Lista de verificación de solicitud de extracción:
> * [ ] Enlace incluido al [problema de GitHub de React Spectrum](https://github.com/adobe/react-spectrum/issues) correspondiente .
> * [ ]  Se agregaron / actualizaron pruebas unitarias y un libro de cuentos para este cambio (para un nuevo código o código que ya tiene pruebas).
> * [ ]  Instrucciones de prueba completas.
> * [ ]  Documentación actualizada (si ya existe para este componente).
> * [ ] Consulte las prácticas de accesibilidad para esta función: [prácticas de Aria](https://www.w3.org/TR/wai-aria-practices-1.1/)
> 
> ##  Instrucciones de prueba:
> ![imagen](https://user-images.githubusercontent.com/1004204/142924791-9950e5c1-076c-44bc-bb9a-3d27d71ff201.png)
> 
> Libro de cuentos: https://reactspectrum.blob.core.windows.net/reactspectrum/c61511f17f3a55ac44dcb2ce9b104373f272fb60/storybook/index.html?path=/story/contextualhelp--default